python_docs_theme/static/pydoctheme.css

```
@@ -328,6 +328,10 @@ tt, code, pre {
     font-size: 96.5%;
 }
 
+pre {
+    line-height: 125%;
```

To keep the initial value for the theme, rather than the overridden one:
```suggestion
    line-height: 120%;
```

---

It was indeed the case. I went for `div.body pre` to increase the specificity and override `pygments_dark.css`